app/core/services/image_manipulation/image_manipulation.py

```diff
@@ -55,17 +66,30 @@ def find_greater_scaled_dimensions(
     original_ratio = original_x / original_y
     new_ratio = requested_x / requested_y
     if new_ratio == original_ratio:
+        log.debug("Same ratio, return requested x and y")
         return requested_x, requested_y
     # If the new ratio is higher, it means that the x grew proportionally
     # more than the y
     if new_ratio > original_ratio:
+        log.debug(
+            "Requested width/height ratio is > than original image width/height ratio. "
+            "Setting requested width and scaling height accordingly keeping the ratio."
+        )
```

Comment:
I would add info about the `width`/`height` requested and the calculated `ratio`, so, if something goes wrong, you can easily replicate the request